src/dfx/src/main.rs

```diff
@@ -50,14 +50,29 @@ pub struct CliOpts {
     command: commands::DfxCommand,
 }
 
-#[derive(Args, Clone, Debug)]
-struct NetworkOpt {
+#[derive(Args, Clone, Debug, Default)]
+pub struct NetworkOpt {
     /// Override the compute network to connect to. By default, the local network is used.
     /// A valid URL (starting with `http:` or `https:`) can be used here, and a special
     /// ephemeral network will be created specifically for this request. E.g.
     /// "http://localhost:12345/" is a valid network name.
-    #[arg(long, global = true)]
+    #[arg(long, global = true, conflicts_with("playground"))]
     network: Option<String>,
+
+    /// Shorthand for --network=playground.
+    /// Borrows short-lived canisters on the real IC network instead of creating normal canisters.
+    #[clap(long, global(true), conflicts_with("network"))]
+    playground: bool,
+}
+
+impl NetworkOpt {
+    pub fn to_network_name(self) -> Option<String> {
```

Comment:
There are two places under src/command/deps that reference NetworkOpt.network directly rather than calling NetworkOpt.to_network_name().  While `--playground` probably wouldn't work with those commands anyway, it seems like an easy mistake to make for future code.  Do you think it would make sense to require all access to `NetworkOpt.network` to go through `NetworkOpt.to_network_name()`?